test/plugins/features/movies/controller.test.js

```diff
@@ -39,4 +48,71 @@ describe('movie controller', () => {
 
   });
 
+  describe('list', () => {
+
+    it('should properly sort based on query params', () => {
+      const query = { sort: 'release_year' , q: '' };
+      const reverseQuery = { sort: '-release_year', q: '' };
+
+      Controller.list(query)
+      .then((movies) => {
+        let ordered = true;
+
+        expect(movies.length).to.be.at.least(1);
+
+        movies.each((movie, index) => {
+          if (!movie.release_year || !movies[index + 1].release_year) {
+            return;
+          }
+
+          if (movie.release_year < movies[index + 1].release_year) {
+            ordered = false;
+          }
+        });
+
+        expect(ordered).to.be.true;
+      });
+
+      return Controller.list(reverseQuery)
+      .then((movies) => {
+        let ordered = true;
+
+        expect(movies.length).to.be.at.least(1);
+
+        movies.each((movie, index) => {
+          if (!movie.release_year || !movies[index + 1].release_year) {
+            return;
+          }
+
+          if (movie.release_year > movies[index + 1].release_year) {
+            ordered = false;
+          }
+        });
+
+        expect(ordered).to.be.true;
+      });
+    });
+
+    it('should filter titles based request query', () => {
+      const q = 'amer';
+      const query = { q, sort: 'release_year' };
+
+      Controller.list(query)
+      .then((movies) => {
+        let filtered = true;
+
+        expect(movies.length).to.be.at.least(1);
```

Comment:
Since you're setting up the db, you can and probably should be more specific about the counts matching: `expect(movies.length).to.eql(1);`. Especially because you want to make sure that movies that didn't match the filter are not returned.